app/views/layouts/application.html.erb

```diff
@@ -30,6 +30,17 @@
     <meta property="og:image" content="/assets/images/govuk-opengraph-image.png">
   </head>
 
+  <!-- Global site tag (gtag.js) - Google Analytics -->
```

Comment:
Needs to be within `<head>` just after the opening `<head>` tag: https://support.google.com/analytics/answer/1008080